controller/controller.go

```diff
@@ -207,6 +227,7 @@ func (c *Controller) bootWithError(ctx context.Context) error {
 		}
 
 		c.logger.LogCtx(ctx, "level", "debug", "message", "ensured custom resource definition exists")
+
```

Comment:
Let's please not add too many empty lines where they do not serve a purpose. Especially when the pattern is not consistently applied.